.github/workflows/test-install.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         runner: [macos-latest, windows-latest]
-        node-version: ['18', '20', '22']
+        node-version: ["18", "20", "22"]
```

Comment:
We'll probably remove `18` with it no longer being supported:
> https://www.herodevs.com/blog-posts/node-js-18-end-of-life-what-it-means-and-how-to-prepare

Reply:
I think we may use 18 on the VMs